lib/index.js

```diff
@@ -1,13 +1,93 @@
-var React = require('react');
-var ReactDOM = require('react-dom');
+const React = require('react');
+const ReactDOM = require('react-dom');
 
-ReactDOM.render(
-  <h1>Hello, world!</h1>,
-  document.getElementById('example')
-);
+import { Router, Route, hashHistory } from 'react-router'
+import { Link } from 'react-router';
 
-new webpack.DefinePlugin({
-  "process.env": {
-    NODE_ENV: JSON.stringify("production")
+import FontIcon from 'material-ui/FontIcon';
+
+require('../css/main.css');
+
+// navbar
+class NavBar extends React.Component{
+  render() {
+    return (
+      <div className="NavBar">
+        GIRLSQUAD
+      </div>
+    );
+  }
+}
+
+// welcome page to login
+class Welcome extends React.Component{
+  constructor(){
+    super();
   }
-});
+
+  render() {
+    return (
+      <div className="grid">
+        <NavBar />
+        <div className="Welcome row">
+          <Link to="/home"><ActionButton id="login-button" text="Login" className="col-5" /></Link>
+          <ActionButton className="col-5" id="signup-button" text="Sign up" />
+        </div>
+      </div>
+    );
+  }
+};
+
+// buttons
+class ActionButton extends React.Component {
```

Comment:
@lingtran Just a question, not a review. Is `extends React.Component {` the way you use react with es6? Is `Component` a React class or one that you created?

 I like how broken out your classes are, and that your naming is explicit. I don't know React but it looks good 👍 .